### DISCUSSION TITLE
Can I disable AdNauseam for local subnet w/o having to disable each site/IP manually?

### DISCUSSION THREAD

**Discussion by TubaOrNotTuba** [dhowe/AdNauseam#1921] [Q&A]
I find AdNauseam blocking functionality of local subnet sites, especially network management sites.  With everyone having at least one of these, their router, but in my case a couple dozen that I use for testing and project building, is there a way to disable AdNauseam from doing anything on any site on the local subnet.  I tried adding 192.168.0.0/23 to the whitelist page but that didn't do any good.  Thoughts?

**Comment by dhowe** [comment]
See https://github.com/dhowe/AdNauseam/issues/1922
